index.html

```diff
@@ -50,7 +53,158 @@ <h2 class="greeting greeting-text headline-box">
           <a href="#" class="social-links"
             ><img src="images/icons/angellist.svg" alt="angellist"
           /></a>
-        </aside>
+        </div>
+      </article>
+
+      <!--My Work section-->
+      <article class="my-work">
```

Comment:
You will agree with me that your projects don't make up an article. Kindly use a `section` tag instead because it is more semantic.